src/antsibull_core/schemas/collection_meta.py

```diff
@@ -112,28 +125,76 @@ def _check_reason_is_not_renamed(self) -> Self:
         return self
 
 
-class CollectionMetadata(p.BaseModel):
+class RemovedRemovalInformation(RemovalInformation):
+    """
+    Stores metadata on when and why a collection was removed.
+    """
+
+    major_version: int
```

Comment:
Might be worth adding a comment that `major_version` cannot be `TBD` here (because it was already removed) like it is in the base class. That tripped me up for a minute. Or we could remove it entirely (and have another `PendingRemovalInformation` sub-model) like I suggested in another comment.

Reply:
Should be no longer necessary due to 47593714d43b0fc8f73d6bbe00ab4f2acb511ae3.